TestCases/parallel_regression_AD.py

```diff
@@ -264,16 +264,16 @@ def main():
     ### Coupled FSI Adjoint         ###
     ###################################
    
-    # Structural model
-    discadj_fsi           = TestCase('discadj_fsi')
-    discadj_fsi.cfg_dir   = "disc_adj_fsi"
-    discadj_fsi.cfg_file  = "configAD_fsi.cfg" 
-    discadj_fsi.test_iter = 3000
-    discadj_fsi.test_vals = [0.958848,-0.157183,0.658415,1.302076] #last 4 columns
-    discadj_fsi.su2_exec  = "parallel_computation.py -f"
-    discadj_fsi.timeout   = 1600
-    discadj_fsi.tol       = 0.00001
-    test_list.append(discadj_fsi)
+#    # Structural model
```

Comment:
@talbring I'll start by saying I hope this was an accident, which is possible given the size of the pull request...

I found out yesterday the FSI discrete adjoint capabilities of the code no longer work, needless to say I am not very happy. I am sure this work could have been done without disrupting the functionalities of the code for people that actively use it, develop it, maintain it, and actively interact with the community.
I could have helped keeping this working while an alternative is not available, I clearly showed interest and availability to help, so, if this was not done accidentally, it is unfair to say the least.

There is no need to revert anything, but for what is worth I will disapprove any release related pull requests until the functionally is put back in the code. I will try to help on #774 put I would have preferred to do so without having to rush.